tests/foreman/cli/test_registration.py

```diff
@@ -0,0 +1,82 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:Assignee: swadeley
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import pytest
+from broker.broker import VMBroker
+
+from robottelo.config import settings
+from robottelo.hosts import ContentHost
+
+
+@pytest.mark.tier1
+def test_upgrade_katello_ca_consumer_rpm(module_org, module_location, default_sat):
+    """After updating the consumer cert the rhsm.conf file still points to Satellite host name
+    and not Red Hat CDN for subscription.
+
+    :id: c8d861ec-0d81-4d89-a8e1-02afecfd8171
+
+    :steps:
+
+        1. Get consumer crt source file
+        2. Install rpm-build
+        3. Use rpmbuild to change the version in the spec file
+        4. Build new RPM with higher version number
+        5. Install new RPM and assert no change in server URL
+
+    :expectedresults: Server URL is still Satellite host name not Red Hat CDN
+
+    :CaseAutomation: Automated
+
+    :CaseImportance: High
```

Comment:
please add `:customerscenario: true` as the associated bz has got rhcp links

Reply:
ACK, change made.

Thank you.